### DISCUSSION TITLE
Medialibrary 7 not working with laravel 8

### DISCUSSION THREAD

**Discussion by vishalinfyom** [spatie/laravel-medialibrary#2133] [Q&A]
PHP Version : 7.3
Laravel: v8.10.0

Giving below error : 

![image](https://user-images.githubusercontent.com/33887755/97267214-ce721c80-184f-11eb-892e-44bf1fcaa631.png)

**Comment by vishalinfyom** [comment]
is it compulsory to use PHP 7.4 with media library 8 ??

**Comment by vishalinfyom** [comment]
Or is it possible to use media library 7 with laravel 8 ?

**Comment by iammart** [comment]
The issue is related to illuminate packages in the composer dependencies only targeting 5, 6 & 7 for the v7 branch.

Im on PHP73 here and have the same issue, ie. unable to install v8 which is compatible with L8 as its dependent on PHP74.

I don't know if @freekmurze would consider updating the composer.json to allow for L8 for the v7 branch?

**Reply by iammart** [reply]
... @vishalinfyom you can refer to @freekmurze comment here https://github.com/spatie/laravel-medialibrary/discussions/2117 on another discussion about the same subject.
